wasm/examples/example_program.c

```diff
@@ -0,0 +1,8 @@
+#include <string.h>
```

Comment:
What do you use this file for?

Reply:
I was originally going to add a section on compiling programs for our runtime from different languages, but it turns out there's no **nice** equivalent of `cargo build --target wasm32-unknown-unknown`.

Directly using `clang` kept popping up errors about not having the right linker and emscripten can only be run in a browser because it uses a non-trivial amount of Javascript glue code. At that point I gave up and removed the section from my blog post. It looks like I forgot to delete `example_program.c`.